src/ape_test/runners.py

```diff
@@ -30,6 +33,61 @@ def _network_choice(self) -> str:
         # The option the user providers via --network (or the default).
         return self.config.getoption("network")
 
+    def pytest_exception_interact(self, report, call):
+        """
+        Called when an exception was raised which can potentially be
+        interactively handled.
+        With the `--interactive` flag, outputs the full repr of the failed test
+        and opens an interactive shell using `brownie._cli.console.Console`.
+        Arguments
```

Comment:
We need to document this method the same we document other methods, like how we list the args and returns type and stuff